Build fails on GCC 11.1.0

Arch Linux x86_64
gcc 11.1.0

```
[parker@wolfcola Sources]$ ./build-linux64.sh 
Will build with 'make -j6' ... please edit this script if incorrect.
+ rm -rf cmake-build
+ mkdir cmake-build
+ cd cmake-build
+ cmake -DCMAKE_BUILD_TYPE=RelWithDebInfo ..
CMake Deprecation Warning at CMakeLists.txt:1 (cmake_minimum_required):
  Compatibility with CMake < 2.8.12 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.


-- The C compiler identification is GNU 11.1.0
-- The CXX compiler identification is GNU 11.1.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Looking for pthread.h
-- Looking for pthread.h - found
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Failed
-- Looking for pthread_create in pthreads
-- Looking for pthread_create in pthreads - not found
-- Looking for pthread_create in pthread
-- Looking for pthread_create in pthread - found
-- Found Threads: TRUE  
-- Found SDL2: /usr/lib/libSDL2main.a;/usr/lib/libSDL2.so;-lpthread  
-- Found ZLIB: /usr/lib/libz.so (found version "1.2.11") 
-- Using assembler code (when available)
-- Not using i386 nasm ASM
-- Configuring done
-- Generating done
-- Build files have been written to: /home/parker/build/Serious-Engine/Sources/cmake-build
+ echo 'ECC first'
ECC first
+ make ecc
[  0%] Building CXX object CMakeFiles/ecc.dir/Ecc/Main.cpp.o
[  0%] Building CXX object CMakeFiles/ecc.dir/Ecc/Parser.cpp.o
[100%] Building CXX object CMakeFiles/ecc.dir/Ecc/Scanner.cpp.o
Ecc/Scanner.cpp:1812:17: warning: ‘void yyunput(int, char*)’ defined but not used [-Wunused-function]
[100%] Linking CXX executable ecc
[100%] Built target ecc
+ echo 'Then the rest...'
Then the rest...
+ make
Consolidate compiler generated dependencies of target ecc
[  1%] Built target ecc
[  1%] Generating ../Engine/Classes/BaseEvents.cpp, ../Engine/Classes/BaseEvents.h, ../Engine/Classes/BaseEvents_tables.h
[  1%] Generating ../Engine/Classes/MovableBrushEntity.cpp, ../Engine/Classes/MovableBrushEntity.h, ../Engine/Classes/MovableBrushEntity_tables.h
[  1%] Generating ../Engine/Classes/MovableEntity.cpp, ../Engine/Classes/MovableEntity.h, ../Engine/Classes/MovableEntity_tables.h
[  2%] Generating ../Engine/Classes/MovableModelEntity.cpp, ../Engine/Classes/MovableModelEntity.h, ../Engine/Classes/MovableModelEntity_tables.h
[  2%] Generating ../Engine/Classes/PlayerEntity.cpp, ../Engine/Classes/PlayerEntity.h, ../Engine/Classes/PlayerEntity_tables.h
[  2%] Generating ../EntitiesMP/AirElemental.cpp, ../EntitiesMP/AirElemental.h, ../EntitiesMP/AirElemental_tables.h
[  2%] Generating ../EntitiesMP/AirShockwave.cpp, ../EntitiesMP/AirShockwave.h, ../EntitiesMP/AirShockwave_tables.h
[  2%] Generating ../EntitiesMP/AmmoItem.cpp, ../EntitiesMP/AmmoItem.h, ../EntitiesMP/AmmoItem_tables.h
[  2%] Generating ../EntitiesMP/AmmoPack.cpp, ../EntitiesMP/AmmoPack.h, ../EntitiesMP/AmmoPack_tables.h
[  2%] Generating ../EntitiesMP/AnimationChanger.cpp, ../EntitiesMP/AnimationChanger.h, ../EntitiesMP/AnimationChanger_tables.h
[  3%] Generating ../EntitiesMP/AnimationHub.cpp, ../EntitiesMP/AnimationHub.h, ../EntitiesMP/AnimationHub_tables.h
[  3%] Generating ../EntitiesMP/AreaMarker.cpp, ../EntitiesMP/AreaMarker.h, ../EntitiesMP/AreaMarker_tables.h
[  3%] Generating ../EntitiesMP/ArmorItem.cpp, ../EntitiesMP/ArmorItem.h, ../EntitiesMP/ArmorItem_tables.h
[  3%] Generating ../EntitiesMP/BackgroundViewer.cpp, ../EntitiesMP/BackgroundViewer.h, ../EntitiesMP/BackgroundViewer_tables.h
[  3%] Generating ../EntitiesMP/BasicEffects.cpp, ../EntitiesMP/BasicEffects.h, ../EntitiesMP/BasicEffects_tables.h
[  3%] Generating ../EntitiesMP/Beast.cpp, ../EntitiesMP/Beast.h, ../EntitiesMP/Beast_tables.h
[  3%] Generating ../EntitiesMP/BigHead.cpp, ../EntitiesMP/BigHead.h, ../EntitiesMP/BigHead_tables.h
[  4%] Generating ../EntitiesMP/BlendController.cpp, ../EntitiesMP/BlendController.h, ../EntitiesMP/BlendController_tables.h
[  4%] Generating ../EntitiesMP/BloodSpray.cpp, ../EntitiesMP/BloodSpray.h, ../EntitiesMP/BloodSpray_tables.h
[  4%] Generating ../EntitiesMP/Boneman.cpp, ../EntitiesMP/Boneman.h, ../EntitiesMP/Boneman_tables.h
[  4%] Generating ../EntitiesMP/Bouncer.cpp, ../EntitiesMP/Bouncer.h, ../EntitiesMP/Bouncer_tables.h
[  4%] Generating ../EntitiesMP/Bullet.cpp, ../EntitiesMP/Bullet.h, ../EntitiesMP/Bullet_tables.h
[  4%] Generating ../EntitiesMP/Camera.cpp, ../EntitiesMP/Camera.h, ../EntitiesMP/Camera_tables.h
[  5%] Generating ../EntitiesMP/CameraMarker.cpp, ../EntitiesMP/CameraMarker.h, ../EntitiesMP/CameraMarker_tables.h
[  5%] Generating ../EntitiesMP/CannonBall.cpp, ../EntitiesMP/CannonBall.h, ../EntitiesMP/CannonBall_tables.h
[  5%] Generating ../EntitiesMP/CannonRotating.cpp, ../EntitiesMP/CannonRotating.h, ../EntitiesMP/CannonRotating_tables.h
[  5%] Generating ../EntitiesMP/CannonStatic.cpp, ../EntitiesMP/CannonStatic.h, ../EntitiesMP/CannonStatic_tables.h
[  5%] Generating ../EntitiesMP/ChainsawFreak.cpp, ../EntitiesMP/ChainsawFreak.h, ../EntitiesMP/ChainsawFreak_tables.h
[  5%] Generating ../EntitiesMP/Copier.cpp, ../EntitiesMP/Copier.h, ../EntitiesMP/Copier_tables.h
[  5%] Generating ../EntitiesMP/Counter.cpp, ../EntitiesMP/Counter.h, ../EntitiesMP/Counter_tables.h
[  6%] Generating ../EntitiesMP/CrateBus.cpp, ../EntitiesMP/CrateBus.h, ../EntitiesMP/CrateBus_tables.h
[  6%] Generating ../EntitiesMP/CrateRider.cpp, ../EntitiesMP/CrateRider.h, ../EntitiesMP/CrateRider_tables.h
[  6%] Generating ../EntitiesMP/CreditsHolder.cpp, ../EntitiesMP/CreditsHolder.h, ../EntitiesMP/CreditsHolder_tables.h
[  6%] Generating ../EntitiesMP/Damager.cpp, ../EntitiesMP/Damager.h, ../EntitiesMP/Damager_tables.h
[  6%] Generating ../EntitiesMP/Debris.cpp, ../EntitiesMP/Debris.h, ../EntitiesMP/Debris_tables.h
[  6%] Generating ../EntitiesMP/DebugEntityStatesDisplay.cpp, ../EntitiesMP/DebugEntityStatesDisplay.h, ../EntitiesMP/DebugEntityStatesDisplay_tables.h
[  6%] Generating ../EntitiesMP/Demon.cpp, ../EntitiesMP/Demon.h, ../EntitiesMP/Demon_tables.h
[  7%] Generating ../EntitiesMP/DestroyableArchitecture.cpp, ../EntitiesMP/DestroyableArchitecture.h, ../EntitiesMP/DestroyableArchitecture_tables.h
[  7%] Generating ../EntitiesMP/Devil.cpp, ../EntitiesMP/Devil.h, ../EntitiesMP/Devil_tables.h
[  7%] Generating ../EntitiesMP/DevilMarker.cpp, ../EntitiesMP/DevilMarker.h, ../EntitiesMP/DevilMarker_tables.h
[  7%] Generating ../EntitiesMP/DevilProjectile.cpp, ../EntitiesMP/DevilProjectile.h, ../EntitiesMP/DevilProjectile_tables.h
[  7%] Generating ../EntitiesMP/DoorController.cpp, ../EntitiesMP/DoorController.h, ../EntitiesMP/DoorController_tables.h
[  7%] Generating ../EntitiesMP/Dragonman.cpp, ../EntitiesMP/Dragonman.h, ../EntitiesMP/Dragonman_tables.h
[  7%] Generating ../EntitiesMP/EffectMarker.cpp, ../EntitiesMP/EffectMarker.h, ../EntitiesMP/EffectMarker_tables.h
[  8%] Generating ../EntitiesMP/Effector.cpp, ../EntitiesMP/Effector.h, ../EntitiesMP/Effector_tables.h
[  8%] Generating ../EntitiesMP/Elemental.cpp, ../EntitiesMP/Elemental.h, ../EntitiesMP/Elemental_tables.h
[  8%] Generating ../EntitiesMP/EnemyBase.cpp, ../EntitiesMP/EnemyBase.h, ../EntitiesMP/EnemyBase_tables.h
[  8%] Generating ../EntitiesMP/EnemyCounter.cpp, ../EntitiesMP/EnemyCounter.h, ../EntitiesMP/EnemyCounter_tables.h
[  8%] Generating ../EntitiesMP/EnemyDive.cpp, ../EntitiesMP/EnemyDive.h, ../EntitiesMP/EnemyDive_tables.h
[  8%] Generating ../EntitiesMP/EnemyFly.cpp, ../EntitiesMP/EnemyFly.h, ../EntitiesMP/EnemyFly_tables.h
[  8%] Generating ../EntitiesMP/EnemyMarker.cpp, ../EntitiesMP/EnemyMarker.h, ../EntitiesMP/EnemyMarker_tables.h
[  9%] Generating ../EntitiesMP/EnemyRunInto.cpp, ../EntitiesMP/EnemyRunInto.h, ../EntitiesMP/EnemyRunInto_tables.h
[  9%] Generating ../EntitiesMP/EnemySpawner.cpp, ../EntitiesMP/EnemySpawner.h, ../EntitiesMP/EnemySpawner_tables.h
[  9%] Generating ../EntitiesMP/EnvironmentBase.cpp, ../EntitiesMP/EnvironmentBase.h, ../EntitiesMP/EnvironmentBase_tables.h
[  9%] Generating ../EntitiesMP/EnvironmentMarker.cpp, ../EntitiesMP/EnvironmentMarker.h, ../EntitiesMP/EnvironmentMarker_tables.h
[  9%] Generating ../EntitiesMP/EnvironmentParticlesHolder.cpp, ../EntitiesMP/EnvironmentParticlesHolder.h, ../EntitiesMP/EnvironmentParticlesHolder_tables.h
[  9%] Generating ../EntitiesMP/Eruptor.cpp, ../EntitiesMP/Eruptor.h, ../EntitiesMP/Eruptor_tables.h
[ 10%] Generating ../EntitiesMP/ExotechLarva.cpp, ../EntitiesMP/ExotechLarva.h, ../EntitiesMP/ExotechLarva_tables.h
[ 10%] Generating ../EntitiesMP/ExotechLarvaBattery.cpp, ../EntitiesMP/ExotechLarvaBattery.h, ../EntitiesMP/ExotechLarvaBattery_tables.h
[ 10%] Generating ../EntitiesMP/ExotechLarvaCharger.cpp, ../EntitiesMP/ExotechLarvaCharger.h, ../EntitiesMP/ExotechLarvaCharger_tables.h
[ 10%] Generating ../EntitiesMP/Eyeman.cpp, ../EntitiesMP/Eyeman.h, ../EntitiesMP/Eyeman_tables.h
[ 10%] Generating ../EntitiesMP/Fireworks.cpp, ../EntitiesMP/Fireworks.h, ../EntitiesMP/Fireworks_tables.h
[ 10%] Generating ../EntitiesMP/Fish.cpp, ../EntitiesMP/Fish.h, ../EntitiesMP/Fish_tables.h
[ 10%] Generating ../EntitiesMP/Flame.cpp, ../EntitiesMP/Flame.h, ../EntitiesMP/Flame_tables.h
[ 11%] Generating ../EntitiesMP/FogMarker.cpp, ../EntitiesMP/FogMarker.h, ../EntitiesMP/FogMarker_tables.h
[ 11%] Generating ../EntitiesMP/Gizmo.cpp, ../EntitiesMP/Gizmo.h, ../EntitiesMP/Gizmo_tables.h
[ 11%] Generating ../EntitiesMP/Global.cpp, ../EntitiesMP/Global.h, ../EntitiesMP/Global_tables.h
[ 11%] Generating ../EntitiesMP/GradientMarker.cpp, ../EntitiesMP/GradientMarker.h, ../EntitiesMP/GradientMarker_tables.h
[ 11%] Generating ../EntitiesMP/GravityMarker.cpp, ../EntitiesMP/GravityMarker.h, ../EntitiesMP/GravityMarker_tables.h
[ 11%] Generating ../EntitiesMP/GravityRouter.cpp, ../EntitiesMP/GravityRouter.h, ../EntitiesMP/GravityRouter_tables.h
[ 11%] Generating ../EntitiesMP/Grunt.cpp, ../EntitiesMP/Grunt.h, ../EntitiesMP/Grunt_tables.h
[ 12%] Generating ../EntitiesMP/GruntSka.cpp, ../EntitiesMP/GruntSka.h, ../EntitiesMP/GruntSka_tables.h
[ 12%] Generating ../EntitiesMP/Guffy.cpp, ../EntitiesMP/Guffy.h, ../EntitiesMP/Guffy_tables.h
[ 12%] Generating ../EntitiesMP/HazeMarker.cpp, ../EntitiesMP/HazeMarker.h, ../EntitiesMP/HazeMarker_tables.h
[ 12%] Generating ../EntitiesMP/Headman.cpp, ../EntitiesMP/Headman.h, ../EntitiesMP/Headman_tables.h
[ 12%] Generating ../EntitiesMP/HealthItem.cpp, ../EntitiesMP/HealthItem.h, ../EntitiesMP/HealthItem_tables.h
[ 12%] Generating ../EntitiesMP/HudPicHolder.cpp, ../EntitiesMP/HudPicHolder.h, ../EntitiesMP/HudPicHolder_tables.h
[ 12%] Generating ../EntitiesMP/Item.cpp, ../EntitiesMP/Item.h, ../EntitiesMP/Item_tables.h
[ 13%] Generating ../EntitiesMP/KeyItem.cpp, ../EntitiesMP/KeyItem.h, ../EntitiesMP/KeyItem_tables.h
[ 13%] Generating ../EntitiesMP/LarvaOffspring.cpp, ../EntitiesMP/LarvaOffspring.h, ../EntitiesMP/LarvaOffspring_tables.h
[ 13%] Generating ../EntitiesMP/Light.cpp, ../EntitiesMP/Light.h, ../EntitiesMP/Light_tables.h
[ 13%] Generating ../EntitiesMP/Lightning.cpp, ../EntitiesMP/Lightning.h, ../EntitiesMP/Lightning_tables.h
[ 13%] Generating ../EntitiesMP/Marker.cpp, ../EntitiesMP/Marker.h, ../EntitiesMP/Marker_tables.h
[ 13%] Generating ../EntitiesMP/MessageHolder.cpp, ../EntitiesMP/MessageHolder.h, ../EntitiesMP/MessageHolder_tables.h
[ 13%] Generating ../EntitiesMP/MessageItem.cpp, ../EntitiesMP/MessageItem.h, ../EntitiesMP/MessageItem_tables.h
[ 14%] Generating ../EntitiesMP/MeteorShower.cpp, ../EntitiesMP/MeteorShower.h, ../EntitiesMP/MeteorShower_tables.h
[ 14%] Generating ../EntitiesMP/MirrorMarker.cpp, ../EntitiesMP/MirrorMarker.h, ../EntitiesMP/MirrorMarker_tables.h
[ 14%] Generating ../EntitiesMP/ModelDestruction.cpp, ../EntitiesMP/ModelDestruction.h, ../EntitiesMP/ModelDestruction_tables.h
[ 14%] Generating ../EntitiesMP/ModelHolder.cpp, ../EntitiesMP/ModelHolder.h, ../EntitiesMP/ModelHolder_tables.h
[ 14%] Generating ../EntitiesMP/ModelHolder2.cpp, ../EntitiesMP/ModelHolder2.h, ../EntitiesMP/ModelHolder2_tables.h
[ 14%] Generating ../EntitiesMP/ModelHolder3.cpp, ../EntitiesMP/ModelHolder3.h, ../EntitiesMP/ModelHolder3_tables.h
[ 15%] Generating ../EntitiesMP/MovingBrush.cpp, ../EntitiesMP/MovingBrush.h, ../EntitiesMP/MovingBrush_tables.h
[ 15%] Generating ../EntitiesMP/MovingBrushMarker.cpp, ../EntitiesMP/MovingBrushMarker.h, ../EntitiesMP/MovingBrushMarker_tables.h
[ 15%] Generating ../EntitiesMP/MusicChanger.cpp, ../EntitiesMP/MusicChanger.h, ../EntitiesMP/MusicChanger_tables.h
[ 15%] Generating ../EntitiesMP/MusicHolder.cpp, ../EntitiesMP/MusicHolder.h, ../EntitiesMP/MusicHolder_tables.h
[ 15%] Generating ../EntitiesMP/NavigationMarker.cpp, ../EntitiesMP/NavigationMarker.h, ../EntitiesMP/NavigationMarker_tables.h
[ 15%] Generating ../EntitiesMP/ParticlesHolder.cpp, ../EntitiesMP/ParticlesHolder.h, ../EntitiesMP/ParticlesHolder_tables.h
[ 15%] Generating ../EntitiesMP/Pendulum.cpp, ../EntitiesMP/Pendulum.h, ../EntitiesMP/Pendulum_tables.h
[ 16%] Generating ../EntitiesMP/PhotoAlbum.cpp, ../EntitiesMP/PhotoAlbum.h, ../EntitiesMP/PhotoAlbum_tables.h
[ 16%] Generating ../EntitiesMP/Pipebomb.cpp, ../EntitiesMP/Pipebomb.h, ../EntitiesMP/Pipebomb_tables.h
[ 16%] Generating ../EntitiesMP/Player.cpp, ../EntitiesMP/Player.h, ../EntitiesMP/Player_tables.h
[ 16%] Generating ../EntitiesMP/PlayerActionMarker.cpp, ../EntitiesMP/PlayerActionMarker.h, ../EntitiesMP/PlayerActionMarker_tables.h
[ 16%] Generating ../EntitiesMP/PlayerAnimator.cpp, ../EntitiesMP/PlayerAnimator.h, ../EntitiesMP/PlayerAnimator_tables.h
[ 16%] Generating ../EntitiesMP/PlayerMarker.cpp, ../EntitiesMP/PlayerMarker.h, ../EntitiesMP/PlayerMarker_tables.h
[ 16%] Generating ../EntitiesMP/PlayerView.cpp, ../EntitiesMP/PlayerView.h, ../EntitiesMP/PlayerView_tables.h
[ 17%] Generating ../EntitiesMP/PlayerWeapons.cpp, ../EntitiesMP/PlayerWeapons.h, ../EntitiesMP/PlayerWeapons_tables.h
[ 17%] Generating ../EntitiesMP/PlayerWeaponsEffects.cpp, ../EntitiesMP/PlayerWeaponsEffects.h, ../EntitiesMP/PlayerWeaponsEffects_tables.h
[ 17%] Generating ../EntitiesMP/PowerUpItem.cpp, ../EntitiesMP/PowerUpItem.h, ../EntitiesMP/PowerUpItem_tables.h
[ 17%] Generating ../EntitiesMP/Projectile.cpp, ../EntitiesMP/Projectile.h, ../EntitiesMP/Projectile_tables.h
[ 17%] Generating ../EntitiesMP/PyramidSpaceShip.cpp, ../EntitiesMP/PyramidSpaceShip.h, ../EntitiesMP/PyramidSpaceShip_tables.h
[ 17%] Generating ../EntitiesMP/PyramidSpaceShipMarker.cpp, ../EntitiesMP/PyramidSpaceShipMarker.h, ../EntitiesMP/PyramidSpaceShipMarker_tables.h
[ 17%] Generating ../EntitiesMP/Reminder.cpp, ../EntitiesMP/Reminder.h, ../EntitiesMP/Reminder_tables.h
[ 18%] Generating ../EntitiesMP/RollingStone.cpp, ../EntitiesMP/RollingStone.h, ../EntitiesMP/RollingStone_tables.h
[ 18%] Generating ../EntitiesMP/Santa.cpp, ../EntitiesMP/Santa.h, ../EntitiesMP/Santa_tables.h
[ 18%] Generating ../EntitiesMP/Scorpman.cpp, ../EntitiesMP/Scorpman.h, ../EntitiesMP/Scorpman_tables.h
[ 18%] Generating ../EntitiesMP/ScrollHolder.cpp, ../EntitiesMP/ScrollHolder.h, ../EntitiesMP/ScrollHolder_tables.h
[ 18%] Generating ../EntitiesMP/SeriousBomb.cpp, ../EntitiesMP/SeriousBomb.h, ../EntitiesMP/SeriousBomb_tables.h
[ 18%] Generating ../EntitiesMP/Ship.cpp, ../EntitiesMP/Ship.h, ../EntitiesMP/Ship_tables.h
[ 18%] Generating ../EntitiesMP/ShipMarker.cpp, ../EntitiesMP/ShipMarker.h, ../EntitiesMP/ShipMarker_tables.h
[ 19%] Generating ../EntitiesMP/Shooter.cpp, ../EntitiesMP/Shooter.h, ../EntitiesMP/Shooter_tables.h
[ 19%] Generating ../EntitiesMP/SoundHolder.cpp, ../EntitiesMP/SoundHolder.h, ../EntitiesMP/SoundHolder_tables.h
[ 19%] Generating ../EntitiesMP/SpawnerProjectile.cpp, ../EntitiesMP/SpawnerProjectile.h, ../EntitiesMP/SpawnerProjectile_tables.h
[ 19%] Generating ../EntitiesMP/Spinner.cpp, ../EntitiesMP/Spinner.h, ../EntitiesMP/Spinner_tables.h
[ 19%] Generating ../EntitiesMP/StormController.cpp, ../EntitiesMP/StormController.h, ../EntitiesMP/StormController_tables.h
[ 19%] Generating ../EntitiesMP/Summoner.cpp, ../EntitiesMP/Summoner.h, ../EntitiesMP/Summoner_tables.h
[ 19%] Generating ../EntitiesMP/SummonerMarker.cpp, ../EntitiesMP/SummonerMarker.h, ../EntitiesMP/SummonerMarker_tables.h
[ 20%] Generating ../EntitiesMP/Switch.cpp, ../EntitiesMP/Switch.h, ../EntitiesMP/Switch_tables.h
[ 20%] Generating ../EntitiesMP/TacticsChanger.cpp, ../EntitiesMP/TacticsChanger.h, ../EntitiesMP/TacticsChanger_tables.h
[ 20%] Generating ../EntitiesMP/TacticsHolder.cpp, ../EntitiesMP/TacticsHolder.h, ../EntitiesMP/TacticsHolder_tables.h
[ 20%] Generating ../EntitiesMP/Teleport.cpp, ../EntitiesMP/Teleport.h, ../EntitiesMP/Teleport_tables.h
[ 20%] Generating ../EntitiesMP/Terrain.cpp, ../EntitiesMP/Terrain.h, ../EntitiesMP/Terrain_tables.h
[ 20%] Generating ../EntitiesMP/TextFXHolder.cpp, ../EntitiesMP/TextFXHolder.h, ../EntitiesMP/TextFXHolder_tables.h
[ 21%] Generating ../EntitiesMP/TimeController.cpp, ../EntitiesMP/TimeController.h, ../EntitiesMP/TimeController_tables.h
[ 21%] Generating ../EntitiesMP/TouchField.cpp, ../EntitiesMP/TouchField.h, ../EntitiesMP/TouchField_tables.h
[ 21%] Generating ../EntitiesMP/Trigger.cpp, ../EntitiesMP/Trigger.h, ../EntitiesMP/Trigger_tables.h
[ 21%] Generating ../EntitiesMP/Twister.cpp, ../EntitiesMP/Twister.h, ../EntitiesMP/Twister_tables.h
[ 21%] Generating ../EntitiesMP/VoiceHolder.cpp, ../EntitiesMP/VoiceHolder.h, ../EntitiesMP/VoiceHolder_tables.h
[ 21%] Generating ../EntitiesMP/Walker.cpp, ../EntitiesMP/Walker.h, ../EntitiesMP/Walker_tables.h
[ 21%] Generating ../EntitiesMP/WatchPlayers.cpp, ../EntitiesMP/WatchPlayers.h, ../EntitiesMP/WatchPlayers_tables.h
[ 22%] Generating ../EntitiesMP/Watcher.cpp, ../EntitiesMP/Watcher.h, ../EntitiesMP/Watcher_tables.h
[ 22%] Generating ../EntitiesMP/Water.cpp, ../EntitiesMP/Water.h, ../EntitiesMP/Water_tables.h
[ 22%] Generating ../EntitiesMP/WeaponItem.cpp, ../EntitiesMP/WeaponItem.h, ../EntitiesMP/WeaponItem_tables.h
[ 22%] Generating ../EntitiesMP/Werebull.cpp, ../EntitiesMP/Werebull.h, ../EntitiesMP/Werebull_tables.h
[ 22%] Generating ../EntitiesMP/Woman.cpp, ../EntitiesMP/Woman.h, ../EntitiesMP/Woman_tables.h
[ 22%] Generating ../EntitiesMP/WorldBase.cpp, ../EntitiesMP/WorldBase.h, ../EntitiesMP/WorldBase_tables.h
[ 22%] Generating ../EntitiesMP/WorldLink.cpp, ../EntitiesMP/WorldLink.h, ../EntitiesMP/WorldLink_tables.h
[ 23%] Generating ../EntitiesMP/WorldSettingsController.cpp, ../EntitiesMP/WorldSettingsController.h, ../EntitiesMP/WorldSettingsController_tables.h
[ 23%] Built target ParseEntities
[ 23%] Building CXX object CMakeFiles/engine_safemath.dir/Engine/Brushes/Brush.cpp.o
In file included from /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/include/x86gprintrin.h:27,
                 from /usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/include/immintrin.h:27,
                 from /usr/include/SDL2/SDL_cpuinfo.h:99,
                 from /usr/include/SDL2/SDL.h:38,
                 from /home/parker/build/Serious-Engine/Sources/./Engine/StdH.h:39,
                 from /home/parker/build/Serious-Engine/Sources/Engine/Brushes/Brush.cpp:16:
/home/parker/build/Serious-Engine/Sources/./Engine/Base/Types.h:235:18: error: ‘ULONG __rold’ redeclared as different kind of entity
  235 |     inline ULONG _rotl(ULONG ul, int bits)
      |                  ^~~~~
/usr/lib/gcc/x86_64-pc-linux-gnu/11.1.0/include/ia32intrin.h:138:1: note: previous declaration ‘unsigned int __rold(unsigned int, int)’
  138 | __rold (unsigned int __X, int __C)
      | ^~~~~~
/home/parker/build/Serious-Engine/Sources/./Engine/Base/Types.h:235:30: error: expected primary-expression before ‘ul’
  235 |     inline ULONG _rotl(ULONG ul, int bits)
      |                              ^~
/home/parker/build/Serious-Engine/Sources/./Engine/Base/Types.h:235:30: error: expected ‘)’ before ‘ul’
/home/parker/build/Serious-Engine/Sources/./Engine/Base/Types.h:235:18: note: to match this ‘(’
  235 |     inline ULONG _rotl(ULONG ul, int bits)
      |                  ^~~~~
In file included from /home/parker/build/Serious-Engine/Sources/./Engine/StdH.h:51,
                 from /home/parker/build/Serious-Engine/Sources/Engine/Brushes/Brush.cpp:16:
/home/parker/build/Serious-Engine/Sources/./Engine/Base/Types.h:700:8: error: ‘__uint64’ does not name a type; did you mean ‘__int64’?
  700 | inline __uint64 BYTESWAP64_unsigned(__uint64 x)
      |        ^~~~~~~~
      |        __int64
In file included from /home/parker/build/Serious-Engine/Sources/./Engine/Math/Vector.h:24,
                 from /home/parker/build/Serious-Engine/Sources/./Engine/Brushes/Brush.h:24,
                 from /home/parker/build/Serious-Engine/Sources/Engine/Brushes/Brush.cpp:18:
/home/parker/build/Serious-Engine/Sources/./Engine/Base/Stream.h:162:20: error: declaration of ‘operator>>’ as non-function
  162 |   inline CTStream &operator>>(__uint64 i) { Read_t( &i, sizeof( i)); BYTESWAP( i); return *this; } // throw char *
      |                    ^~~~~~~~
/home/parker/build/Serious-Engine/Sources/./Engine/Base/Stream.h:162:28: error: expected ‘;’ at end of member declaration
  162 |   inline CTStream &operator>>(__uint64 i) { Read_t( &i, sizeof( i)); BYTESWAP( i); return *this; } // throw char *
      |                            ^~
      |                              ;
/home/parker/build/Serious-Engine/Sources/./Engine/Base/Stream.h:162:39: error: expected ‘)’ before ‘i’
  162 |   inline CTStream &operator>>(__uint64 i) { Read_t( &i, sizeof( i)); BYTESWAP( i); return *this; } // throw char *
      |                              ~        ^~
      |                                       )
/home/parker/build/Serious-Engine/Sources/./Engine/Base/Stream.h:174:20: error: declaration of ‘operator<<’ as non-function
  174 |   inline CTStream &operator<<(__uint64 i) { BYTESWAP( i); Write_t( &i, sizeof( i)); return *this; } // throw char *
      |                    ^~~~~~~~
/home/parker/build/Serious-Engine/Sources/./Engine/Base/Stream.h:174:28: error: expected ‘;’ at end of member declaration
  174 |   inline CTStream &operator<<(__uint64 i) { BYTESWAP( i); Write_t( &i, sizeof( i)); return *this; } // throw char *
      |                            ^~
      |                              ;
/home/parker/build/Serious-Engine/Sources/./Engine/Base/Stream.h:174:39: error: expected ‘)’ before ‘i’
  174 |   inline CTStream &operator<<(__uint64 i) { BYTESWAP( i); Write_t( &i, sizeof( i)); return *this; } // throw char *
      |                              ~        ^~
      |                                       )
make[2]: *** [CMakeFiles/engine_safemath.dir/build.make:76: CMakeFiles/engine_safemath.dir/Engine/Brushes/Brush.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:94: CMakeFiles/engine_safemath.dir/all] Error 2
make: *** [Makefile:136: all] Error 2
```

--- a/Sources/Engine/Base/CTString.cpp
+++ b/Sources/Engine/Base/CTString.cpp
@@ -291,7 +291,7 @@ ULONG CTString::GetHash(void) const
   INDEX len = strlen(str_String);
 
   for(INDEX i=0; i<len; i++) {
-    ulKey = _rotl(ulKey,4)+toupper(str_String[i]);
+    ulKey = rotl(ulKey,4)+toupper(str_String[i]);
   }
   return ulKey;
 }

--- a/Sources/Engine/Base/Types.h
+++ b/Sources/Engine/Base/Types.h
@@ -232,7 +232,7 @@ MY_STATIC_ASSERT(size_tSize, sizeof(size_t) == sizeof(void*));
         }
     }
 
-    inline ULONG _rotl(ULONG ul, int bits)
+    inline ULONG rotl(ULONG ul, int bits)
     {
         #if (defined __GNU_INLINE_X86_32__)
             // This, on the other hand, is wicked fast.  :)

